services/src/main/java/com/broadleafcommerce/subscriptionoperation/service/modification/AbstractModifySubscriptionHandler.java

```diff
@@ -42,15 +55,22 @@
  * @author Nathan Moore (nathandmoore)
  */
 @Slf4j
+@RequiredArgsConstructor
 public abstract class AbstractModifySubscriptionHandler implements ModifySubscriptionHandler {
 
+    private static final SecureRandom SECURE_RANDOM = new SecureRandom();
+
+    @Getter(value = AccessLevel.PROTECTED, onMethod_ = @Nullable)
```

Comment:
Not sure why this would be nullable.
```suggestion
    @Getter(value = AccessLevel.PROTECTED)
```